07-bootstrap-validation.md

```diff
@@ -56,18 +56,19 @@ GitOps allows a team to author Kubernetes manifest files, persist them in their
 > Mindful Timing for Upgrades:
 > - Be mindful of when upgrades should occur. If you have overlapping maintenance windows, AKS will determine the running order.
 > - To avoid conflicts, leave at least 24 hours between maintenance window configurations. The timing will depend on the number of nodes in your specific cluster and the duration required for upgrades.
+> - Current configuration should not represent a conflict since Kubernetes version updates are applied manually when customer see fit.
```

Comment:
```suggestion
> - Current configuration should not represent a conflict since AKS version updates are applied manually when the cluster operator sees fit.
```